stac_fastapi/elasticsearch/tests/conftest.py

```diff
@@ -153,7 +153,10 @@ def app_client(api_client, load_test_data):
     try:
         client.create_collection(coll, request=MockStarletteRequest)
     except ConflictError:
-        pass
+        try:
```

Comment:
these tests do not clean up well, especially when any of them fail. This is a stop-gap until that's better handled.